types/muicss/index.d.ts

```diff
@@ -0,0 +1,243 @@
+// Type definitions for muicss React bindings v09.36
+// Project: https://www.muicss.com/
+// Definitions by: Samuel Neff <https://github.com/samuelneff>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.6
+
+/// <reference types="react" />
+
+declare module "muicss/react" {
+    export import Appbar = MuicssReact.Appbar;
+    export import Button = MuicssReact.Button;
+    export import Checkbox = MuicssReact.Checkbox;
+    export import Col = MuicssReact.Col;
+    export import Container = MuicssReact.Container;
+    export import Divider = MuicssReact.Divider;
+    export import Dropdown = MuicssReact.Dropdown;
+    export import DropdownItem = MuicssReact.DropdownItem;
+    export import Form = MuicssReact.Form;
+    export import Input = MuicssReact.Input;
+    export import Option = MuicssReact.Option;
+    export import Panel = MuicssReact.Panel;
+    export import Radio = MuicssReact.Radio;
+    export import Row = MuicssReact.Row;
+    export import Select = MuicssReact.Select;
+    export import Tab = MuicssReact.Tab;
+    export import Tabs = MuicssReact.Tabs;
+    export import Textarea = MuicssReact.Textarea;
+}
+
+declare module 'muicss/lib/react/appbar' {
+    export import Appbar = MuicssReact.Appbar;
+    export default Appbar;
+}
+
+declare module 'muicss/lib/react/button' {
```

Comment:
Instead of using `declare module`, you should reflect these in the package structure.